docs/setup.md

```diff
@@ -27,16 +27,12 @@ cd cyclone
 docker-compose -f docker-compose.yml up -d
 ```
 
-- Using Kubectl: this approach requires a few yaml files. You can read related files in cirlce/scripts/k8s for more detial. Follow these instructions:
+- Using Kubectl: this approach requires a few yaml files. You can read related files in cyclone/scripts/k8s for more detial, and be sure to substitute the values of parameter with real values according to your own situation before executing these instructions:
```

Comment:
and adjust the parameters before executing these instructions

Reply:
done